lib/sedna/algorithms/optical_flow/__init__.py

```diff
@@ -40,6 +37,9 @@ def __call__(self, old_frame=None, current_frame=None):
 
 @ClassFactory.register(ClassType.OF, alias="LukasKanadeOF")
 class LukasKanade(BaseFilter, abc.ABC):
+    import cv2
+    import numpy
```

Comment:
`numpy` is s the basic dependency of Sedna.

Reply:
OK, I placed the `numpy` import inside the class because I noticed that it is imported at the function level in the `FedAvg` class. So I thought about following the same approach.

Anyway, I will move the import up to the module level.